### ISSUE TITLE
Комментарии к заявке

### ISSUE THREAD

**Issue by aakuptsov** [Ineptum/typist#1]
1. >>Получение информации о правильной постановке рук на клавиатуре и о советах по развитию навыка

Каким образом это будет осуществляться? Продукт будет анализировать данные с камеры?

2. >>Тренировка навыка печати и набор текста

Необходимо сказать несколько слов о том, как именно будет проходить тренировка. Это можно сделать в разделе "проблемное поле". 

3. Не слишком ли много - и Mac OS-версия, и Windows?

**Comment by Ineptum** [comment]
1. Нет, получение информации пользователем будет осуществляться путём того, что в приложении ему будет показана картинка, наглядно иллюстрирующая, как правильно расставлять пальцы на клавиатуре и каким пальцем какую клавишу нажимать.

2. Хорошо.

3. Не слишком! Всё успеется!

**Comment by aakuptsov** [comment]
Зачёт, но

комментарий от Салимова Д. Р.:

зачёт, но проблемы с оригинальностью: похожих приложений много, надо расширить раздел 'аналогичные продукты'. Например, есть ещё популярный сервис 'соло на клавиатуре' или 'клавогонки'.

**Comment by Ineptum** [comment]
Так, я исследовал вопрос. Думаю, с идеей соревнований в реальном времени в клавиатурном тренажёре не выйдет, так как для её реализации мне придётся дополнительно освоить технологию socket'ов и websocket'ов, её реализации через Python и Swift вместе. На это уйдёт очень много времени, учитывая, что я собираюсь реализовать своё приложение как на macOS, так и на Windows. Думаю, я бы мог этим заняться, только теперь мне нужно спросить у Вас разрешение перенести разработку исключительно под macOS

**Comment by aakuptsov** [comment]
@Ineptum Правильно ли я понимаю, что проблема только в том, что вместо двух платформ ты хочешь использовать одну - Mac OS?

**Comment by Ineptum** [comment]
Да. Всё верно, @aakuptsov.

**Comment by aakuptsov** [comment]
@Ineptum Ок, можешь делать только для одной платформы :)

**Comment by Ineptum** [comment]
@aakuptsov Мне всё ещё нужно успеть в некоторый срок расписать сценарии для соревнований в реальном времени или я могу сделать это позже?
